src/tests/system/tests/test_ipa.py

```diff
@@ -1015,3 +1015,23 @@ def test_ipa__hbac_user_host_service_category_equals_all(client: Client, ipa: IP
     client.sssd.restart()
     assert not client.auth.ssh.password("user1", "Secret123"), "user1 should be denied after restriction!"
     assert not client.auth.ssh.password("user2", "Secret123"), "user2 should be denied after restriction!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__prohibited_short_names(client: Client, ipa: IPA):
+    """
+    :title: SSSD refuses to start with short names on IPA
+    :setup:
+        1. Set full_name_format to %1$s on IPA
+    :steps:
+        1. Restart SSSD on IPA
+    :expectedresults:
+        1. SSSD must refuse to start
+    :customerscenario: False
+    """
+    cfg = "[domain/ipa.test]\nfull_name_format = %1$s\n"
```

Comment:
Hi,

the IPA domain might not be called `ipa.test` in all test environment, I guess it would be better to use `ipa.domain` instead or the framework functions which can modify sssd.conf directly.

Is there a reason for using a config snippet?

bye,
Sumit

Reply:
Good point, updated.

Regarding snippets - yes it is needed here. The main config is somehow overwritten/managed by topology. I did not look into details tough. This was the easiest way to set it.